.github/workflows/test.yml

```diff
@@ -31,4 +32,12 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Run e2e tests
-        run: make test-e2e
+        run: make test-e2e-release
+        working-directory: ${{ github.workspace }}
+
+      - name: Upload Cypress screenshots (on failure)
```

Comment:
this is a smidge scary, as the workflow will execute with secrets and they'd be being uploaded publicly(?)

Reply:
We can remove the screenshots feature if you prefer?

Reply:
I think I would, but needs to be weighed up by how useful they are. inclined to say if you need a screenshot then run it locally

Reply:
Sounds reasonable to me. Another option could be to do something like this:

```
    volumes:
      - ../web:/web
      - ../web/cypress/screenshots:/web/cypress/screenshots
```

So that the screenshots for the release build are available locally, and during CI they don't get uploaded as artefacts. What do you reckon?